packages/core/src/node/mdx/loader.ts

```diff
@@ -133,31 +137,30 @@ export default async function mdxLoader(
 
   try {
     let compileResult: string;
+    let pageMeta = { title: '', toc: [] } as PageMeta;
+
     if (!enableMdxRs) {
-      const mdxOptions = await createMDXOptions(
-        docDirectory,
-        config,
-        checkDeadLinks,
-        routeService,
-        filepath,
-        pluginDriver,
+      const compiler = createProcessor(
+        await createMDXOptions(
+          docDirectory,
+          config,
+          checkDeadLinks,
+          routeService,
+          filepath,
+          options.pluginDriver,
```

Comment:
can also be destructed from `const { config, docDirectory, checkDeadLinks, routeService } = options;`